categories/checkboxes/exclusive-checkbox.md

```diff
@@ -0,0 +1,43 @@
+# Configure an Exclusive Checkbox Option in SurveyJS
```

Comment:
I think we better add a demo, seeing that we now have a Checkboxes category:

https://surveyjs.io/form-library/examples/create-checkboxes-question-in-javascript/